src/main/java/org/folio/des/scheduling/base/BaseExportJobScheduler.java

```diff
@@ -60,6 +61,11 @@ public List<Job> scheduleExportJob(ExportConfig exportConfig) {
     return scheduledJobs;
   }
 
+  @Override
+  public void deleteJobGroup(String tenantId) {
+    throw new UnsupportedOperationException("Not implemented yet");
```

Comment:
let's just log here, so that old scheduling implementation is not affected

Reply:
I will fix, Thanks